docs/_sass/_colors.scss

```diff
@@ -7,3 +7,10 @@ $color-gray3: #C6C9CA;
 $color-green: #07d973;
 $color-pink: #F68DFE;
 $color-white: #FFFFFF;
+
+$color-light-grey-green: #C9D3C5;
```

Comment:
Rather than giving these specific color names, why not just use something like we have on OldDot where we just use the actual purpose of the colors? Like this:

<img width="336" alt="image" src="https://user-images.githubusercontent.com/2319350/209191921-da387293-a44f-4fa5-ab03-26535a9bfc55.png">

Reply:
I think we should have them be color names because we may want to use them for multiple purposes.

Reply:
This is similar to what we do on the rest of New Dot (and for the same reason): https://github.com/Expensify/App/blob/main/src/styles/colors.js#L21

Reply:
For example, we use `darker-green` for the background of the lhn and the background of the cards. We could name it, `dark-green-background` or something like that, but then we'd want to adjust that name again when applying it elsewhere. 

Instead, the colors can be abstract, and the css style selectors can be where the functional meaning comes across. 

Does that make sense?